containers/sarek/Dockerfile

```diff
@@ -1,18 +1,9 @@
-FROM debian:8.9
+FROM nfcore/base:latest
 
 LABEL \
-  author="Maxime Garcia" \
+  authors="Maxime.Gracia@scilifelab.se, Szilveszter.Juhos@scilifelab.se" \
   description="Image with tools used in Sarek" \
-  maintainer="maxime.garcia@scilifelab.se"
+  maintainers="Maxime.Gracia@scilifelab.se, Szilveszter.Juhos@scilifelab.se"
```

Comment:
If I remember well, we can use whichever label we want, but `maintainer` is meant to stay that way, because it's a port of the deprecated instruction MAINTAINER.
I do think we can use:
```
maintainer="Maxime Garcia <maxime.garcia@scilifelab.se>, Szilveszter Juhos <Szilveszter.Juhos@scilifelab.se>"
```